www/js/onboarding/PrivacyPolicy.tsx

```diff
@@ -148,15 +167,7 @@ const PrivacyPolicy = () => {
       <Text>{'\n'}</Text>
       <Text style={styles.text}>
         {t('consent-text.rights.destroy-data-pt1')}
-        {/* Linking is broken, look into enabling after migration
-                    <Text
-                        style={styles.hyperlinkStyle(colors.primary)}
-                        onPress={() => {
-                            Linking.openURL("mailto:k.shankari@nrel.gov");
-                        }}>
-                        k.shankari@nrel.gov
-                    </Text> */}
-        <Text style={styles.text}>(k.shankari@nrel.gov)</Text>
+        <Text>{'(k.shankari@nrel.gov)'}</Text>
```

Comment:
does inappbrowser (IAB) not work for mailto: links?

Reply:
No, I'm not sure that it does work, I've tried adding a `mailto:` link, but it just shows the link instead being a link:

<img width="304" alt="Screenshot 2024-01-16 at 3 12 52 PM" src="https://github.com/e-mission/e-mission-phone/assets/54848919/4bca8719-2b47-4101-816a-163e75f698ae">

Reply:
Update - I got a link to show, but I can't click it

Reply:
I tried with the devapp on my LG phone, in case it was an emulator quirk, but link is still not tappable, so no, I don't think mailto links work with this plugin